### PR TITLE
chore(dependencies)-bump non-breaking, code cleanup

### PR DESCRIPTION
**Pull request by AllenBW**
Updates ui-components and pfa to the latest on master

☝️ but a bitta time passed, so now this updates angular, and cleans up a few things related to that change

### CODE REVIEW

**Comment by himdel** [comment]
Iiinteresting :) The fact that this even says `/home/himdel` means something is definitely wrong.

..Yes, sorry about that, 1.1.13 is broken, looks like another case of me forgetting to make sure prepublish succeeds before releasing :(.

Fixed in 1.1.14.

File: client/app/core/config.js [hunk 1]
```diff
@@ -12,10 +12,6 @@ const hasDevTools = angular.isDefined(window.__REDUX_DEVTOOLS_EXTENSION__)
 export function configure ($logProvider, $compileProvider, $qProvider, $ngReduxProvider) {
   $logProvider.debugEnabled(isDevel)
   $compileProvider.debugInfoEnabled(isDevel)
-
-  // TODO: Remove following line as per: https://docs.angularjs.org/guide/migration#migrate1.5to1.6-ng-services-$compile
-  $compileProvider.preAssignBindingsEnabled(true)
```
[the remark this answers is not in the document]
Comment (AllenBW) [reply client/app/core/config.js]: woah woah woah... 14 days of silence and NOW you pop in 😆 yeah am working that presently 😏

[the remark this answers is not in the document]
Comment (himdel) [reply client/app/core/config.js]: Heheh, miq bot woke me up :) 

Ok, perfect, looking forward to it :) :+1:

**Comment by himdel** [comment]
Just to be sure... components not using onInit:

```
$ ag '\.component\(' | sed -e 's/.*, //' -e 's/)//' | while read foo; do ag "export.*$foo" </dev/tty | cut -d: -f1 ; done | while read bar; do grep -q onInit $bar || echo $bar; done

client/app/orders/request-list/requests-list.component.js  # no controller
client/app/services/custom-button/custom-button.component.js  # no init code
client/app/services/custom-button-menu/custom-button-menu.component.js  # no init code
client/app/services/service-details/service-details-ansible-modal.component.js  # init code does not rely on bindings
client/app/services/usage-graphs/usage-graphs.component.js # using $onChanges
client/app/core/shopping-cart/shopping-cart.component.js # init code does not rely on bindings
client/app/shared/icon-list/icon-list.component.js # no controller
client/app/shared/icon-status.component.js # using $onChanges
client/app/shared/loading.component.js  # no init code
client/app/shared/ss-card/ss-card.component.js # no controller
client/app/shared/tagging/tagging.component.js # init code does not rely on bindings
client/app/shared/timeline/timeline.component.js  # init code does not rely on bindings
client/app/components/dashboard/dashboard.component.service.js # false match
client/app/services/custom-button/custom-button.component.js  # no init code
client/app/services/service-details/service-details-ansible-modal.component.js # init code does not rely on bindings
client/app/services/usage-graphs/usage-graphs.component.js # using $onChanges
```

:+1:

**Comment by himdel** [comment]
So.. the changes look good, not seeing any breakage in the UI once logged in, I'd say LGTM :).

Execpt I'm also seeing this:

wut

**Comment by himdel** [comment]
Created https://github.com/patternfly/patternfly/issues/1057

Maybe they'll reply this was intentional, but @terezanovotna is not aware of any such changes having been planned, so maybe not.

**Pull request merged**